src/sentry/static/sentry/app/views/projectPlugins.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import AsyncView from './asyncView';
+import {t} from '../locale';
+import ExternalLink from '../components/externalLink';
+import Checkbox from '../components/checkbox';
+import IndicatorStore from '../stores/indicatorStore';
+
+export default class ProjectPlugins extends AsyncView {
+  getTitle() {
+    return 'Project Integration Settings';
+  }
+
+  getEndpoints() {
+    let {projectId, orgId} = this.props.params;
+    return [['plugins', `/projects/${orgId}/${projectId}/plugins/`]];
+  }
+
+  handleChange(pluginId, shouldEnable, idx) {
+    let {projectId, orgId} = this.props.params;
+
+    let method = shouldEnable ? 'POST' : 'DELETE';
+
+    let loadingIndicator = IndicatorStore.add(t('Saving changes..'));
+
+    this.api.request(`/projects/${orgId}/${projectId}/plugins/${pluginId}/`, {
+      method,
+      success: () => {
+        let plugins = this.state.plugins.slice();
+        plugins[idx].enabled = shouldEnable;
+        this.setState({
```

Comment:
The checkbox when toggling enable feels pretty laggy. Since it's a controlled component, it's waiting until after this API request finishes before updating its value.